exp/api/v1beta1/azuremanagedmachinepool_webhook.go

```diff
@@ -193,8 +215,20 @@ func (r *AzureManagedMachinePool) validateLastSystemNodePool(cli client.Client)
 }
 
 func (r *AzureManagedMachinePool) validateMaxPods() error {
-	if to.Int32(r.Spec.MaxPods) < 10 || to.Int32(r.Spec.MaxPods) > 250 {
-		return errors.New("MaxPods must be between 10 and 250")
+	if r.Spec.MaxPods != nil {
```

Comment:
This (and the updates to UT) will be cleaned up once https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1915 merges

Reply:
Unnecessary foo no longer present